pages/linux/protonvpn-cli.md

```diff
@@ -2,7 +2,7 @@
 
 > Official ProtonVPN client.
```

Comment:
I think we should change the description. This rewrite, although widely used, does not seem like an officially-endorsed client. I could be wrong though; if that's the case, sources please.